Products/ZenUtils/zenpack.py

```diff
@@ -528,10 +532,20 @@ def _linkedRestore(self, zp):
         with open(os.devnull, 'w') as fnull:
             subprocess.check_call(cmd, stdout=fnull, stderr=fnull)
 
-    def _restore(self, zenpackID, zenpackVersion, filesOnly):
-        # if the version has a dash, replace with an underscore
-        zenpackVersion = zenpackVersion.replace("-", "_", 1)
-        # look for the egg
+    def _loadFiles(self, zenpackID, zenpackVersion):
+        """
+        Install all zenpacks with '--files-only' flag for preventing
+        DistributionNotFound Error.
+        """
+        zenpack = self._findEggs(zenpackID, zenpackVersion)[0]
+        if zenpack and zenpack.lower().endswith(".egg"):
+            self.log.info("Loading files for %s", zenpack)
+            cmd = ["zenpack", "--files-only", "--install", zenpack]
+            subprocess.check_call(cmd)
```

Comment:
Since no database work is required here, is it possible to just directly call the code that performs the files-only installs rather than running the zenpack command in subshell?